src/frontends/pytorch/src/op/add.cpp

```diff
@@ -28,6 +29,16 @@ OutputVector translate_add_common(const NodeContext& context, bool inplace) {
         // Case when two lists gets concatenated
         PYTORCH_OP_CONVERSION_CHECK(false, "aten::add is used for concatenation of lists, not possible to convert");
     }
+
+    if (lhs.get_element_type() == element::boolean && rhs.get_element_type() == element::boolean) {
```

Comment:
Please also check dtype with `NodeContext::get_input_type()`. Like is done in:
https://github.com/openvinotoolkit/openvino/blob/fa43c7b364b6899a90da2b3a27d77461235a5071/src/frontends/pytorch/src/op/sum.cpp#L46

Reply:
done